sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,89 @@
+import atexit
+from multiprocessing import Process
+import numpy as np
+import platform
+from queue import Empty, Queue
+import tensorflow as tf
+from threading import Thread
+
+from rllab.sampler.utils import rollout
+
+__all__ = ["Plotter"]
+
+
+class Plotter(object):
+    def __init__(self, env, policy, sess=None, graph=None):
+        self.env = env
+        self.policy = policy
+        self.sess = tf.get_default_session() if sess is None else sess
+        self.graph = tf.get_default_graph() if graph is None else graph
+        self.worker_thread = Thread(target=self._start_worker, daemon=True)
+        self.queue = Queue()
+
+    def _start_worker(self):
+        env = None
+        policy = None
+        max_length = None
+        initial_rollout = True
+        try:
+            with self.sess.as_default(), self.sess.graph.as_default():
```

Comment:
Can you add a comment here explaining that the worker processes all messages in the queue per loop, not one message per loop?